app/src/main/java/sdp/moneyrun/database/GameDbData.java

```diff
@@ -60,6 +63,56 @@ public GameDbData(String name,
         this.isStarted = false;
     }
 
+    public GameDbData(String name,
+                      Player host,
+                      List<Player> players,
+                      int maxPlayerCount,
+                      Location startLocation,
+                      boolean isVisible,
+                      List<Coin> coins,
+                      int numCoins,
+                      double radius,
+                      double duration){
+        if(name == null){
+            throw new IllegalArgumentException("name should not be null.");
+        }
+        if(host == null){
+            throw new IllegalArgumentException("host should not be null.");
+        }
+        if(players == null){
+            throw new IllegalArgumentException("players should not be null.");
+        }
+        if(startLocation == null){

```

Comment:
Easy test case that can improve coverage

Reply:
Yes